packages/expo-web-browser/android/build.gradle

```diff
@@ -52,6 +52,10 @@ android {
   lintOptions {
     abortOnError false
   }
+  compileOptions {
+    sourceCompatibility = '1.8'
+    targetCompatibility = '1.8'
+  }
```

Comment:
Do we need this here?
Moreover we have it here: https://github.com/expo/expo/blob/764dbd1fe0a4dda34c1f4f101a7231a20b82a6c2/android/app/build.gradle#L24

Reply:
Yep, but that's another module. If we want to use lambdas in here, we need that. Android Studio supports lambdas in 3.0+, but needs that in modules.